pkgs/stdenv/adapters.nix

```diff
@@ -90,7 +94,6 @@ rec {
                     name = name + "-" + cross.config;
                     nativeBuildInputs = nativeBuildInputsDrvs
                       ++ nativeInputsFromBuildInputs
-                      ++ [ gccCross binutilsCross ]
```

Comment:
I mean this adapter function takes `gccCross` and `binutilsCross` as args, but do we need them anymore passed that way?